test-framework/src/main/java/io/quarkus/vault/test/VaultTestExtension.java

```diff
@@ -201,7 +207,9 @@ public void start() throws Exception {
 
         postgresContainer.setPortBindings(Arrays.asList(MAPPED_POSTGRESQL_PORT + ":" + POSTGRESQL_PORT));
 
-        rabbitMQContainer = new RabbitMQContainer()
+        // 3.7.25-management-alpine is the default version in RabbitMQContainer but the constants is private
```

Comment:
should we access it through reflection? if not we are going to have to make sure we stay aligned (and it is probably going to be forgotten).

Reply:
That's a really old, unsupported version of RabbitMQ. Community support for RMQ 3.13 just ended. Maybe we should just be pestering TestContainers to get off their asses 🤣